lib/cognitivebing.rb

```diff
@@ -4,6 +4,8 @@
 require 'net/http'
 
 class CognitiveBing
+  class BingUnavailableException < Exception; end
```

Comment:
inherit from standarderror and not from exception

Reply:
http://blog.honeybadger.io/ruby-exception-vs-standarderror-whats-the-difference/